ts/features/payments/checkout/types/PspPaymentMethodNotAvailableProblemJson.ts

```diff
@@ -0,0 +1,33 @@
+import * as t from "io-ts";
+import { enumType } from "@pagopa/ts-commons/lib/types";
+
+export enum FaultCodeCategoryEnum {
+  "PAYMENT_METHOD_NOT_AVAILABLE_ERROR" = "PAYMENT_METHOD_NOT_AVAILABLE_ERROR"
```

Comment:
What do you think to change also the naming of this enum value?

```suggestion
  "PSP_PAYMENT_METHOD_NOT_AVAILABLE_ERROR" = "PSP_PAYMENT_METHOD_NOT_AVAILABLE_ERROR"
```

Reply:
Changes addressed in this [commit](https://github.com/pagopa/io-app/pull/6393/commits/dedf5f8bcc31877fe58c544a73eff136fe9a2c01)